handlers/deploy.go

```diff
@@ -142,12 +142,27 @@ func makeDeploymentSpec(request types.FunctionDeployment, existingSecrets map[st
 	annotations := buildAnnotations(request)
 
 	var serviceAccount string
+	var volumeName string
```

Comment:
You'd have to handle updates too, if someone wanted to remove a volume that was already attached.